src/components/NFTPortfolio/TopUsersList.vue

```diff
@@ -20,11 +20,11 @@
           @mouseenter.native.once="onHover(user.index)"
         >
           <LinkV2
-            :to="localeLocation({
+            :to="user.id ? localeLocation({
```

Comment:
`user.id` is the `key` for `v-for` though, should we actually skip item without `id` in data?